client-src/elements/chromedash-featurelist.ts

```diff
@@ -8,48 +10,47 @@ const MAX_FEATURES_SHOWN = 500;
 class ChromedashFeaturelist extends LitElement {
   static styles = FEATURELIST_CSS;
 
-  static get properties() {
-    return {
-      user: {type: Object},
-      isSiteEditor: {type: Boolean},
-      signedInUser: {type: String},
-      editableFeatures: {type: Object},
-      features: {attribute: false}, // Directly edited and accessed in template/features.html
-      metadataEl: {attribute: false}, // The metadata component element. Directly edited in template/features.html
-      searchEl: {attribute: false}, // The search input element. Directly edited in template/features.html
-      filtered: {attribute: false},
-      openFeatures: {attribute: false},
-      starredFeatures: {attribute: false},
-    };
-  }
-
-  constructor() {
-    super();
-    this.user = {};
-    this.features = [];
-    this.editableFeatures = [];
-    this.filtered = [];
-    this.metadataEl = document.querySelector('chromedash-metadata');
+  @property({type: Object})
+  user!: User;
+  @property({type: Boolean})
+  isSiteEditor = false;
+  @property({type: String})
+  signedInUser = '';
+  @property({type: Array})
+  editableFeatures!: Feature[];
+  @property({type: Object})
+  metadataEl; // The metadata component element. Directly edited in template/features.html
+  @property({attribute: false})
+  _onFeatureToggledBound;
+  @property({attribute: false})
+  _onStarToggledBound;
+  @state()
+  features!: Feature[]; // Directly edited and accessed in template/features.html
+  @state()
+  searchEl; // The search input element. Directly edited in template/features.html
+  @state()
+  filtered!: Feature[];
+  @state()
+  openFeatures = new Set<number>();
+  @state()
+  starredFeatures = new Set<number>();
+  @state()
+  _hasInitialized = false; // Used to check initialization code.
+  @state()
+  _hasScrolledByUser = false; // Used to set the app header state.
+  @state()
+  _featuresUnveilMetric = new Metric('features_unveil');
+  @state()
+  _featuresFetchMetric = new Metric('features_loaded');
```

Comment:
Similarly, these weren't [reactive](https://lit.dev/docs/components/properties/#internal-reactive-state) before your change, so you shouldn't add reactivity in your change.
```suggestion
  _hasInitialized = false; // Used to check initialization code.
  _hasScrolledByUser = false; // Used to set the app header state.
  _featuresUnveilMetric = new Metric('features_unveil');
  _featuresFetchMetric = new Metric('features_loaded');
```